Backend/.env.example

```
@@ -43,4 +43,8 @@ Chatbot__Ollama__EmbeddingModel=your_ollama_embedding_model_here
 # Gemini Settings
 Chatbot__Gemini__Model=gemini-2.5-flash
 Chatbot__Gemini__ApiKey=your_gemini_api_key_here
-Chatbot__Gemini__EmbeddingModel=text-embedding-004
\ No newline at end of file
+Chatbot__Gemini__EmbeddingModel=text-embedding-004
```

dotenv-linter (4.0.0)

[warning] [LowercaseKey] The Chatbot__Gemini__EmbeddingModel key should be in uppercase

(LowercaseKey)